{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/Error-Implementations"},"kind":"article","schemaVersion":{"patch":0,"major":0,"minor":3},"topicSections":[{"identifiers":["doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/localizedDescription"],"title":"Instance Properties","generated":true,"anchor":"Instance-Properties"}],"metadata":{"roleHeading":"API Collection","role":"collectionGroup","title":"Error Implementations","modules":[{"name":"Pulse"}]},"sections":[],"variants":[{"paths":["\/documentation\/pulse\/remotelogger\/connectionerror\/error-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/com.github.pulse\/documentation\/Pulse","doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger","doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError"]]},"references":{"doc://com.github.pulse/documentation/Pulse/RemoteLogger":{"navigatorTitle":[{"kind":"identifier","text":"RemoteLogger"}],"url":"\/documentation\/pulse\/remotelogger","identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger","type":"topic","fragments":[{"kind":"keyword","text":"class"},{"text":" ","kind":"text"},{"text":"RemoteLogger","kind":"identifier"}],"kind":"symbol","title":"RemoteLogger","abstract":[{"type":"text","text":"Connects to the remote server and sends logs remotely. In the current version,"},{"text":" ","type":"text"},{"type":"text","text":"a server is a Pulse Pro app for macOS)."}],"role":"symbol"},"doc://com.github.pulse/documentation/Pulse/RemoteLogger/ConnectionError/localizedDescription":{"role":"symbol","kind":"symbol","identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError\/localizedDescription","url":"\/documentation\/pulse\/remotelogger\/connectionerror\/localizeddescription","type":"topic","title":"localizedDescription","abstract":[],"fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"localizedDescription"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"}]},"doc://com.github.pulse/documentation/Pulse/RemoteLogger/ConnectionError":{"navigatorTitle":[{"kind":"identifier","text":"ConnectionError"}],"type":"topic","kind":"symbol","abstract":[],"title":"RemoteLogger.ConnectionError","role":"symbol","identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/RemoteLogger\/ConnectionError","url":"\/documentation\/pulse\/remotelogger\/connectionerror","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ConnectionError"}]},"doc://com.github.pulse/documentation/Pulse":{"title":"Pulse","url":"\/documentation\/pulse","abstract":[{"text":"Logger and network inspector for Apple platforms.","type":"text"}],"role":"collection","type":"topic","identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse","kind":"symbol"}}}